dist/index.js

```diff
@@ -98,29 +98,17 @@ class RelayCompilerWebpackPlugin {
   apply(compiler) {
     var _this = this;
 
-    compiler.plugin('before-compile', (() => {
-      var _ref = _asyncToGenerator(function* (compilationParams, callback) {
-        try {
-          const runner = new _relayCompiler.Runner({
-            parserConfigs: _this.parserConfigs,
-            writerConfigs: _this.writerConfigs,
-            reporter: _this.reporter,
-            onlyValidate: false,
-            skipPersist: true
-          });
-
-          yield runner.compileAll();
-        } catch (error) {
-          callback(error);
-          return;
-        }
-        callback();
+    compiler.plugin('compile', _asyncToGenerator(function* () {
```

Comment:
I'm not sure about compile being the correct phase we want to inject too, however `compile` is a sync hook not async. As you can tell there's no `callback` passed in. So we can't safely run any async code safely at this stage.